apps/robots-generator/package.json

```diff
@@ -0,0 +1,50 @@
+{
+  "name": "robots-generator",
+  "version": "0.1.0",
+  "private": true,
+  "scripts": {
+    "dev": "next dev",
+    "build": "next build",
+    "start": "next start",
+    "lint": "next lint"
```

Comment:
Some scripts are missing here e.g. `clean`, `-check`, etc. Ensure it matches what's in the other apps.